requirements.txt

```diff
@@ -7,3 +7,7 @@ jinja2
 PyYAML
 cryptography
 packaging
+# NOTE: resolvelib 0.x version bumps should be considered major/breaking
+# NOTE: and we should update the upper cap with care, at least until 1.0
+# NOTE: Ref: https://github.com/sarugaku/resolvelib/issues/69
+resolvelib >= 0.5.3, < 0.6.0  # New dependency resolver that pip>=20.3 uses
```

Comment:
```suggestion
resolvelib >= 0.5.3, < 0.6.0  # dependency resolver used by ansible-galaxy
```

Reply:
@mattclay I wanted to mention that it's the same as pip uses

Reply:
That's fine to mention in the PR or docs, but why is it relevant in the requirements file?

Reply:
fair enough